### DISCUSSION TITLE
Test Inter with Square Punctuation

### DISCUSSION THREAD

**Discussion by rsms** [rsms/inter#613] [General]
[https://d.rsms.me/tmp/Inter-4.0-7477b2b85a.zip](https://d.rsms.me/tmp/Inter-4.0-7477b2b85a.zip)

![einrst8re](https://github.com/rsms/inter/assets/47046/8491e5fd-e9d7-41e2-a170-9c86c8d3bc07)

![sqpunc-400](https://github.com/rsms/inter/assets/47046/cd538d17-76b3-4931-8aab-d7ca81d9da15)

Related issue: #178

**Comment by Gabre-Rod** [comment]
This is gonna make this font an ABSOLUTE BANGER.

**Comment by AlfinGusdaputra** [comment]


**Reply by rsms** [reply]
Very strange. What software are you using? Can you please provide detail steps to reproduce this? Thank you

**Reply by AlfinGusdaputra** [reply]
It's done sir, i'm sorry honestly I think this is my fault,  because the problem is that the old font file is still overwritten, so it is unreadable for the new version 😅🙏.
